pipelines-in-pipelines/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/tektoncd/experimental/pipelines-in-pipelines
+
+go 1.15
+
+require (
+	github.com/google/go-cmp v0.5.4
+	github.com/hashicorp/go-multierror v1.1.0
+	github.com/tektoncd/pipeline v0.20.0
+	go.opencensus.io v0.22.5
+	go.uber.org/zap v1.16.0
+	google.golang.org/api v0.36.0
+	k8s.io/api v0.20.2
+	k8s.io/apimachinery v0.18.12
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
+	knative.dev/pkg v0.0.0-20210114223020-f0ea5e6b9c4e
+)
+
+// Knative deps
+replace (
+	contrib.go.opencensus.io/exporter/stackdriver => contrib.go.opencensus.io/exporter/stackdriver v0.12.9-0.20191108183826-59d068f8d8ff
+	github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v38.2.0+incompatible
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.4.0+incompatible
+)
+
+// Pin k8s deps to v0.18.8
+replace (
+	k8s.io/api => k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.18.8
+	k8s.io/client-go => k8s.io/client-go v0.18.8
+	k8s.io/code-generator => k8s.io/code-generator v0.18.8
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
+)
```

Comment:
Do we need these replace? 
It's not a problem if we do, just something that we'd need to maintain in future.

Reply:
yes yes, otherwise I hit some issues like this https://github.com/kubernetes/client-go/issues/741 

we use the same deps in cel custom tasks, so can rework them separately